galaxy_ng/tests/unit/api/test_api_ui_sync_config.py

```diff
@@ -103,10 +115,13 @@ def test_negative_update_community_repo_data_without_requirements_file(self):
                 "name": "community",
                 "policy": "immediate",
                 "requirements_file": None,
-                "url": "https://galaxy.ansible.com",
+                "url": "https://galaxy.ansible.com/v3/collections",
```

Comment:
Could you add a note in the PR description the fix/change on the tests related to this PR?

Reply:
@awcrosby pulp_ansible started to validate URLs so this PR was failing with:
```
pulp: galaxy_ng.tests.unit.api.test_api_ui_sync_config:DEBUG: response.data: {'errors': [{'status': '400', 'code': 'invalid', 'title': 'Invalid input.', 'detail': "Invalid URL https://galaxy.ansible.com. Ensure the URL ends in either '/v2/collections' or '/v3/collections'"}]}
```